src/args_mod.f

```diff
@@ -1,12 +1,12 @@
 C> @file
-C
-C> Wrapper for routines iargc and getarg
-C> @author Iredell @date 98-11-DD
+C> @brief Wrapper for routines iargc and getarg.
+C> @author Mark Iredell @date 1998-11-DD
 C>
```

Comment:
We need a totally blank line to separate the file-level doxygen block from the module level. So:
```
C> @brief Wrapper for routines iargc and getarg.
C> @author Mark Iredell @date 1998-11-DD

C> This Fortran Module acts as a wrapper to the system
...
```